Tests/test_Affy.py

```diff
@@ -82,75 +86,99 @@ def testAffy3(self):
 
             global message
             try:
-                numpy.testing.assert_allclose(record.intensities,
-                                              [[234.0, 170.0, 22177.0, 164.0, 22104.0],
-                                               [188.0, 188.0, 21871.0, 168.0, 21883.0],
-                                               [188.0, 193.0, 21455.0, 198.0, 21300.0],
-                                               [188.0, 182.0, 21438.0, 188.0, 20945.0],
-                                               [193.0, 20370.0, 174.0, 20605.0, 168.0]])
+                numpy.testing.assert_allclose(
```

Comment:
I don't know how you're handling the numpy matrices when applying black styling. If preferred, I can make another commit turning fmt: off for the matrices.

Reply:
These look fine - we do have concerns when the layout hides some meaning. Specifically small integer matrices can get squashed into one line, when there are much better shown one line per row.

Reply:
See discussion on #3089 about this and https://github.com/biopython/biopython/pull/3089/commits/cb065f6ffe3f13bf988df5a4fb07f99793a4547f

Reply:
Ok! Thanks for clarifying.